src/utils/OpenSwathWorkflow.cpp

```diff
@@ -901,6 +901,38 @@ class TOPPOpenSwathWorkflow
     }
     std::sort(sw_windows.begin(), sw_windows.end(), SortPairDoubleByFirst);
 
+    // (iv) Estimate min_upper_edge_dist from swathes
+    if (sonar) // We skip this step in SONAR-mode
```

Comment:
@hroest In SONAR-mode, we don't require min_upper_edge_dist at all, right? Is there any other preferred behavior?